vnext/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -316,16 +296,18 @@ void ViewViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate, const folly
       const std::string& propertyName = pair.first.getString();
       const folly::dynamic& propertyValue = pair.second;
 
-      if (TryUpdateBackgroundBrush(*pPanel, propertyName, propertyValue))
+      if (propertyName == "backgroundColor")
```

Comment:
It would be nice to update the property helper to handle this case, though that's definitely not needed for this check-in.